app/providers/atmosphere/optimization_strategy/cpu_load.rb

```diff
@@ -0,0 +1,35 @@
+module Atmosphere
+  module OptimizationStrategy
+    class CpuLoad < OptimizationStrategy::Default
+      def monitor_vms?
+        true
+      end
+
+      def handle_event(hash)
+        logger.info "Received complex event: #{hash}"
+      end
+
+      def event_definitions
+        time_window = 300
+        interval = 600
+        {
+          simple_event: {
+            name: 'CpuLoad',
+            properties: {
+              'vmUuid': 'String',
+              'cpuLoad': 'float'
+            }
+          },
+          complex_event:
+            "select avg(cpuLoad), vmUuid"\
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.